GitUI/UserControls/RevisionGrid/ChangeCount.cs

```diff
@@ -0,0 +1,83 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Text;
+using GitCommands;
+
+namespace GitUI
+{
+    public sealed class ChangeCount
```

Comment:
Is it for artificial commits? Then should we call it something like `ArtificialCommitGitItemStatus`?

Reply:
OK
I had the intention to not change this code when refactoring, it is a pain following changes. Will make a separate commit (including IvanGreks suggestions).
`ArtificialCommitChaneCount`